src/pages/JobDescribtion/components/JobDescriptionItem.jsx

```diff
@@ -0,0 +1,45 @@
+import { useTheme } from "@emotion/react";
+import {
+  ToggleButton,
+  Grid,
+  ToggleButtonGroup,
+  ListItem,
+  ListItemText,
+} from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+function JobDescriptionItem(props) {
+  const [selected, setSelected] = React.useState(() => [0]);
+  const handleChange = (_, newSelected) => {
+    setSelected(newSelected);
+  };
```

Comment:
I think with this approach you can not use that state in the Page itself. You should pass the state to this component from the Page.